CHANGELOG.rst

```diff
@@ -9,7 +9,4 @@ v0.0.1
 
 **Changed:**
 
-* Fixed rever GH address
-
-
-
+* Fixed rever GH address
```

Comment:
I think this should be a bugfix not a change.  Changes are for changed functionality

Reply:
This was what was already there, I can add an additional line for a bug fix

Reply:
I don't believe the changelog has been maintained in keeping up with PRs